tidy3d/components/grid/grid_spec.py

```diff
@@ -136,6 +136,78 @@ def _add_pml_to_bounds(num_layers: Tuple[int, int], bounds: Coords1D) -> Coords1
         add_right = bounds[-1] + last_step * np.arange(1, num_layers[1] + 1)
         return np.concatenate((add_left, bounds, add_right))
 
+    @staticmethod
+    def _postprocess_unaligned_grid(
+        axis: Axis,
+        structures: List[StructureType],
```

Comment:
it looks like only `structures[0].geometry` is needed in this function, is that correct? I think it would make sense to replace `structures: List[StructureType]` with `simulation_box: Box`